src/main/java/net/mcreator/ui/views/editor/image/action/ImageEditorRedoAction.java

```diff
@@ -30,8 +30,7 @@
 	public ImageEditorRedoAction(ActionRegistry actionRegistry) {
 		super(actionRegistry, L10N.t("action.image_editor.redo"), actionEvent -> {
 			JPanel pan = actionRegistry.getMCreator().mcreatorTabs.getCurrentTab().getContent();
-			if (pan instanceof ImageMakerView) {
-				ImageMakerView codeEditorView = (ImageMakerView) pan;
+			if (pan instanceof ImageMakerView codeEditorView) {

```

Comment:
Seems this was copied from the code editor originally, maybe rename codeEditorView to imageMakerView too while at it, same for files below